dbt/adapters/presto/connections.py

```diff
@@ -128,9 +103,10 @@ def _escape_value(cls, value):
 
         I think "'" (a single quote) is the only character that matters.
         """
+        NUMBERS = (decimal.Decimal, int, float)
```

Comment:
let's just name this `numbers` - all-caps is usually reserved for globals

Reply:
Changed: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/11170a8129dc850ff76f76047206293892b0ea03